front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -345,7 +358,7 @@ const SimulationReportSheet = ({
         </View>
         {mapCanvas && (
           <View style={styles.map.map} id="simulationMap">
-            <Image src={mapCanvas} />
+            {mapBlob && <Image src={URL.createObjectURL(mapBlob)} />}
```

Comment:
Seems like this allocates resources each time the component is re-rendered: https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL_static#memory_management

Suggestions:

- Call `URL.createObjectURL()` from the `useEffect()` callback.
- Use [URL.revokeObjectURL()](https://developer.mozilla.org/en-US/docs/Web/API/URL/revokeObjectURL_static) when overwriting a previous URL with a fresh one

Reply:
done :)